flink-runtime-web/web-dashboard/src/app/pages/job/job-local.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+import { combineLatest, Observable, ReplaySubject } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+
+import { JobDetailCorrect, NodesItemCorrect } from 'interfaces';
+
+type Nullable<T> = T | null;
```

Comment:
This is only used once, not shorter than what it encapsulates, and adds an indirection to an unexported type in a public API – I would remove this type and just write `| null`.